### DISCUSSION TITLE
Is there an option to set `collapse` globally for callouts?

### DISCUSSION THREAD

**Discussion by dense-set** [quarto-dev/quarto-cli#9225] [Q&A]
### Description

I am writing solutions to exercises (for a blogpost for my website) inside the [`.callout-tip` divs](https://quarto.org/docs/authoring/callouts.html#markdown-syntax). It's great that there is an option to have these expandable and collapsed by default:
```qmd
:::{.callout-tip collapse="true"} 
... 
:::
```
I was wondering if there was an option akin to `callout-appearance` that can be set in the yaml of the page (or the project) that sets the desired collapse behavior globally without the need to specify it for each callout div.

**Comment by mcanouil** [comment]
- https://github.com/quarto-dev/quarto-cli/discussions/2869

**Reply by cderv** [reply]
FWIW I believe the solution there is not working anymore because 1.3 introduced Custom AST nodes for Callout.

I have made an example with the new Lua to use for such example 
https://examples.quarto.pub/collapse-callout/
